nixos/modules/installer/cd-dvd/installation-cd-base.nix

```diff
@@ -52,5 +52,13 @@
 
   programs.git.enable = lib.mkDefault true;
 
+  # Provide networkmanager for easy network configuration.
+  networking.networkmanager = {
+    enable = true;
+    # Ship without the VPN plugins
+    enableDefaultPlugins = false;
+  };
+  networking.wireless.enable = lib.mkImageMediaOverride false;
```

Comment:
What’s turning this on for us to need to disable it?

Reply:
I don't know, I copied it from graphical. Maybe @ElvishJerricco knows?

Reply:
`nixos/modules/profiles/installation-device.nix` sets it. Let’s fix that (and the advice in `services.getty.helpLine`) and remove the overrides from the other installer modules.

Reply:
In particular this whole config should probably just move there.

Reply:
Updated, thanks.

Reply:
Ok, replaced that with a disable for nm.